### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by muhammedfurkan** [community/community#19061] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/9547040/174857739-68b184ef-5680-41b3-b2b6-11986ffc73ae.png)

**Comment by supermavster** [comment]
Hi the same problem here, the solution:

1. Go to [https://github.com/github-copilot/tp_signup](https://github.com/github-copilot/tp_signup)
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select **Allow** or **Block**.
4. Press `Save and get started`
5. Close or Reload the IDE or Code Editor

Happy Codding ;)

**Comment by NikolayN-ru** [comment]
try restarting IDE or switch from chrome https://github.com/github-copilot/signup?editor=copilot_1.30.616
<img width="386" alt="Снимок экрана 2022-06-22 в 00 09 31" src="https://user-images.githubusercontent.com/67255397/174869345-37f51d63-89f5-459b-a211-8b8d83faee67.png">

**Comment by davecheney** [comment]
Thank you. This issue has been resolved.

**Reply by supermavster** [reply]
because my solution is "This comment was marked as off topic."? solve the problem in another way

> Hi the same problem here, the solution:
> 
> 1. Go to https://github.com/github-copilot/tp_signup
> 2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
> 3. In Suggestions matching public code; select **Allow** or **Block**.
> 4. Press `Save and get started`
> 5. Close or Reload the IDE or Code Editor
> 
> Happy Codding ;)

**Reply by gdonald** [reply]
> because my solution is "This comment was marked as off topic."? solve the problem in another way
> 
> > Hi the same problem here, the solution:
> > 
> > 1. Go to https://github.com/github-copilot/tp_signup
> > 2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
> > 3. In Suggestions matching public code; select **Allow** or **Block**.
> > 4. Press `Save and get started`
> > 5. Close or Reload the IDE or Code Editor
> > 
> > Happy Codding ;)

This worked for me.  Not sure why people think it's off-topic.

**Reply by supermavster** [reply]
@gdonald Thanks for say that!

**Comment by gdonald** [comment]
I have the same problem.  I've tried everything, agree to billing and no billing info, restart vscode, logout of Github.  Nothing works for me.

**Reply by julioxavierr** [reply]
Yeah, the issue still persists for me. Also tried everything.

**Reply by chainyo** [reply]
Same here!

**Reply by 201flaviosilva** [reply]
Same

**Reply by BirdMachine** [reply]
Samesies

**Reply by cyaconi** [reply]
Same

**Reply by DeanFromLondon** [reply]
Same for me, Github support didn't solve the issue

**Reply by wangShengCool** [reply]
same for me,what can t try?
